### ISSUE TITLE
Ice Box Station - Game breaking shaft miner issue 

### ISSUE THREAD

**Issue by Pollnarev** [tgstation/tgstation#77475]
## Round ID:

[212106](https://scrubby.melonmesa.com/round/212106)

## Reproduction:

This issue is specific to Ice Box Station and happened to me on 2 separate occasions.

1. Load up on your cool miner gear
2. head out to mine
3. find and slip on bananium ore or get frozen by an ice drake
4. congratulations, you can no longer interact with any object outside of your inventory, and if you drop something you can no longer pick it up

**Comment by newfren** [comment]
did you have your multi-z quality set to low? I know this fucks up a LOT of things on lower z-levels

**Comment by Pollnarev** [comment]
> did you have your multi-z quality set to low? I know this fucks up a LOT of things on lower z-levels

Yeah I do, it seems to screw with mesons somehow on rare occasions on Icebox and some other z-level maps, I've never had this issue on Metastation
